tests/util/test_material_settings_parser.py

```diff
@@ -0,0 +1,67 @@
+from octoprint_mrbeam import parse_csv
+from octoprint_mrbeam.util.device_info import (
+    MODEL_MRBEAM_2_DC,
+    MODEL_MRBEAM_2_DC_x,
```

Comment:
this still needs to be 'MODEL_MRBEAM_2_DC_X'

Reply:
I would say even if the naming convention is small x but as a representation it still make sense for this to be capitalized

Reply:
Yes, for the constants it makes sense, I missed this one